systemvm/debian/opt/cloud/bin/setup/cloud-early-config

```diff
@@ -89,6 +89,12 @@ start() {
   rm -f /root/.rnd
   echo "" > /root/.ssh/known_hosts
 
+  if which growpart > /dev/null; then
+    growpart /dev/vda 2
+    growpart /dev/vda 6
+    resize2fs /dev/vda6
```

Comment:
@ustcweizhou have you tested this for KVM, and non-KVM hypervisors? It's possible the partitions are not called (hard-coded) vda; instead of this can be find/use partitions by mount point names or other valid/static identifiers?

Reply:
This is screenshot of cpvm on xenserver for example:
![Screenshot from 2020-09-29 10-58-27](https://user-images.githubusercontent.com/95203/94516228-f9faea80-0242-11eb-96a9-8e43415a944d.png)

Reply:
Screenshot of partitions of cpvm on kvm:
![Screenshot from 2020-09-29 11-01-32](https://user-images.githubusercontent.com/95203/94516323-32022d80-0243-11eb-8f9d-9c7562d29878.png)

Reply:
Screenshot of partitions of cpvm, on vmware:
![Screenshot from 2020-09-29 11-03-39](https://user-images.githubusercontent.com/95203/94516459-768dc900-0243-11eb-8681-ae4174102ad4.png)

Reply:
> @ustcweizhou have you tested this for KVM, and non-KVM hypervisors? It's possible the partitions are not called (hard-coded) vda; instead of this can be find/use partitions by mount point names or other valid/static identifiers?

@rhtyd good point. thanks for review